app/services/idv/gpo_mail.rb

```diff
@@ -41,39 +43,26 @@ def last_not_too_recent_enabled?
       IdentityConfig.store.minimum_wait_before_another_usps_letter_in_hours != 0
     end
 
-    attr_reader :current_user
-
     def too_many_letter_requests_within_window?
       return false unless window_limit_enabled?
-
-      number_of_letter_requests_within(
-        IdentityConfig.store.max_mail_events_window_in_days.days,
-        maximum: IdentityConfig.store.max_mail_events,
-      ) >= IdentityConfig.store.max_mail_events
+      current_user.gpo_confirmation_codes.where(
+        updated_at: Range.new(
+          IdentityConfig.store.max_mail_events_window_in_days.days.ago,
+          Time.zone.now,
+        ),
```

Comment:
We might be able to use the open-ended range syntax here right? unless we had been stubbing "now" in the specs?
```suggestion
        updated_at: IdentityConfig.store.max_mail_events_window_in_days.days.ago..,
```

Reply:
I've always found that syntax a little tricky. We pulled it out in this change but it is easy enough to put it back without any problems too.

Reply:
the `Range.new` jumped out at me, I feel like we use `..` or `...` syntax more, if we want to laeve in explicit `Time.zone.now` but switch to `..` I'm cool with that

Reply:
@jmhooper had actually recommended the explicit Range syntax. I don't feel strongly but will let you two battle it out. ;)

(I secretly think the open-ended range syntax was a design misstep in the language and it's easy to miss, but I thought that about the implicit-begin syntax when rescuing within a method too, and would prefer to be consistent with the rest of the app than to enforce my random opinions.)

Reply:
I'll defer to you two, this is not that important to me